src/ng/http.js

```diff
@@ -1293,7 +1301,7 @@ function $HttpProvider() {
           reqHeaders[(config.xsrfHeaderName || defaults.xsrfHeaderName)] = xsrfValue;
         }
 
-        $httpBackend(config.method, url, reqData, done, reqHeaders, config.timeout,
+        $httpBackend(config.method, needsTrustedUrl ? $sce.trustAsResourceUrl(url) : url, reqData, done, reqHeaders, config.timeout,
```

Comment:
If we are going with not securing the params then I can do a tidy refactor on this PR that removes the need for this `trustAs` call